packages/store/abi/ERC165.sol/ERC165.abi.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
I added `.gitattributes` in another PR but might be worth adding here to land it sooner to suppress these codegen diffs for now:
https://github.com/latticexyz/mud/pull/1354/files#diff-618cd5b83d62060ba3d027e314a21ceaf75d36067ff820db126642944145393e